charts/cp-kafka-connect/values.yaml

```diff
@@ -24,6 +24,8 @@ servicePort: 8083
 ## ref: https://docs.confluent.io/current/connect/userguide.html#configuring-workers
 connectKeyConverter: "io.confluent.connect.avro.AvroConverter"
 connectValueConverter: "io.confluent.connect.avro.AvroConverter"
+connectKeyConverterSchemaEnable: true
```

Comment:
can we have this disabled by default?

Reply:
Yea sure we can the only thing was that this was enabled by default atm so i kept it that way, but will make a change to make it false by default